app/views/locations/new.html.erb

```diff
@@ -45,6 +45,10 @@
                                            :name,
                                            hint: { text: "This includes stairlifts, ramps, level-access showers or grab rails" },
                                            legend: { text: "Are the majority of units in this location built or adapted to wheelchair-user standards?", size: "m" } %>
+      
+      <%= f.govuk_date_field :startdate,
+                              legend: { text: "When did the first property in this location become available?", size: "m" },
```

Comment:
@Dan13L3 should this text be:

`When did the first property in this location become available under this scheme?`

I'm wondering if there might be cases where the property has been available for social let but not with the supported services previously that might be ambiguous/confusing without additional clarification?